2024-04-04 13:10:18	ERROR	Ping error: API Report Error: ServiceUnavailable(API Report Error: ServiceUnavailable)

System.Exception: API Report Error: ServiceUnavailable
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
2024-04-04 13:11:27	ERROR	Ping error: API Report Error: ServiceUnavailable(API Report Error: ServiceUnavailable)

System.Exception: API Report Error: ServiceUnavailable
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
2024-04-04 13:12:28	ERROR	Ping error: API Report Error: ServiceUnavailable(API Report Error: ServiceUnavailable)

System.Exception: API Report Error: ServiceUnavailable
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
2024-04-04 13:13:37	ERROR	Ping error: API Report Error: ServiceUnavailable(API Report Error: ServiceUnavailable)

System.Exception: API Report Error: ServiceUnavailable
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
2024-04-04 13:14:46	ERROR	Ping error: API Report Error: ServiceUnavailable(API Report Error: ServiceUnavailable)

System.Exception: API Report Error: ServiceUnavailable
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
2024-04-04 17:07:31	ERROR	Failed loading Task(Этот хост неизвестен. (mine.qubic.li:443))

System.Net.Http.HttpRequestException: Этот хост неизвестен. (mine.qubic.li:443)
 ---> System.Net.Sockets.SocketException (11001): Этот хост неизвестен.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError, CancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16)
   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|281_0(AwaitableSocketAsyncEventArgs, ValueTask, CancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String, Int32, HttpRequestMessage, Boolean, CancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String, Int32, HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.AddHttp2ConnectionAsync(QueueItem)
   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken)
   at System.Net.Http.HttpConnectionPool.HttpConnectionWaiter`1.WaitForConnectionAsync(Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage, Boolean, Boolean, CancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage, HttpCompletionOption, CancellationTokenSource, Boolean, CancellationTokenSource, CancellationToken)
   at li.qubic.trainer.Client.GetTrainingSeed()
   at li.qubic.trainer.Trainer.LoadMiningSeed()
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String, Int32, HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.AddHttp2ConnectionAsync(QueueItem)
   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken)
   at System.Net.Http.HttpConnectionPool.HttpConnectionWaiter`1.WaitForConnectionAsync(Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage, Boolean, Boolean, CancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage, HttpCompletionOption, CancellationTokenSource, Boolean, CancellationTokenSource, CancellationToken)
   at li.qubic.trainer.Client.GetTrainingSeed()
   at li.qubic.trainer.Trainer.LoadMiningSeed()
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError, CancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16)
   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|281_0(AwaitableSocketAsyncEventArgs, ValueTask, CancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String, Int32, HttpRequestMessage, Boolean, CancellationToken)
2024-04-04 17:07:32	ERROR	Ping error: Этот хост неизвестен. (mine.qubic.li:443)(Этот хост неизвестен. (mine.qubic.li:443))

System.Net.Http.HttpRequestException: Этот хост неизвестен. (mine.qubic.li:443)
 ---> System.Net.Sockets.SocketException (11001): Этот хост неизвестен.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError, CancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16)
   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|281_0(AwaitableSocketAsyncEventArgs, ValueTask, CancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String, Int32, HttpRequestMessage, Boolean, CancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String, Int32, HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.AddHttp2ConnectionAsync(QueueItem)
   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken)
   at System.Net.Http.HttpConnectionPool.HttpConnectionWaiter`1.WaitForConnectionAsync(Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage, Boolean, Boolean, CancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage, HttpCompletionOption, CancellationTokenSource, Boolean, CancellationTokenSource, CancellationToken)
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String, Int32, HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.AddHttp2ConnectionAsync(QueueItem)
   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken)
   at System.Net.Http.HttpConnectionPool.HttpConnectionWaiter`1.WaitForConnectionAsync(Boolean, CancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage, Boolean, Boolean, CancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage, Boolean, CancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage, HttpCompletionOption, CancellationTokenSource, Boolean, CancellationTokenSource, CancellationToken)
   at li.qubic.trainer.Client.Ping(ReportRequest pingRequest)
   at li.qubic.trainer.Trainer.PingApi()
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError, CancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16)
   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|281_0(AwaitableSocketAsyncEventArgs, ValueTask, CancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String, Int32, HttpRequestMessage, Boolean, CancellationToken)
